extensions/amp-viewer-integration/0.1/test/integration/test-messaging-handshake.js

```diff
@@ -108,7 +108,7 @@ describes.sandboxed('AmpViewerMessagingIntegration', {}, () => {
       });
 
       it('should perform polling handshake', function () {
-        this.timeout(5000);
+        this.timeout(10000);
```

Comment:
It has been a long-standing goal to reduce the timeouts in our tests (#19765). Curious: Is there a way to induce this polling handshake to happen faster during tests by say, using a fake timer?